vtex-apps/modal-layout
# Fix/modal jump

#### What problem is this solving?

Modal moving to the right when you open it and the container has a scroll

#### How should this be manually tested?

Open the [Workspace](https://modaljump--storecomponents.myvtex.com/) and open a quickview
Do the same in [master](https://storecomponents.myvtex.com/) to see the problem (remember that if you're using macos you have to keep showing the scrollbar to see the problem)

#### Checklist/Reminders


#### Screenshots or example usage

#### Type of changes


✔️ | Type of Change
---|---
✔️ | Bug fix 
_ | New feature 
_ | Breaking change 
✔️ | Technical improvements 

#### Notes
